test/integration/contract.js

```diff
@@ -215,12 +215,11 @@ describe('Contract', function () {
 
       // preclaim
       const { salt: _salt } = await contract.aensPreclaim(name)
```

Comment:
the `aensPreclaim` should be replace with the contract call and delegation signature.